incident.py

```diff
@@ -5,10 +5,9 @@
 @task
 @roles('class-cache')
 def fail_to_mirror():
-    """Fails the site to the mirror"""
+    """Fails the site to the mirror by stopping nginx on the cache nodes"""
     puppet.disable("Fabric fail_to_mirror task invoked")
-    nginx.disable_vhost("www.gov.uk")
-    nginx.force_restart()
+    nginx_kill()
```

Comment:
I don't think there's any way this command can work. It's a function that's not defined anywhere.